src/e2e-test/features/dataplex/sink/DesignTimeDataplex.feature

```diff
@@ -0,0 +1,91 @@
+@Dataplex_Sink
+Feature: Dataplex Sink - Design time scenarios
+
+  @BATCH-TS-DATAPLEX-DSGN-01
+  Scenario: To verify user should be able to validate mandatory fields in dataplex sink when asset type is selected as BQ dataset
+    When Open Datafusion Project to configure pipeline
+    And Select Sink plugin: "Dataplex" from the plugins list
+    And Navigate to the properties page of plugin: "Dataplex"
+    And Enter input plugin property: "dataplexReferenceName" with value: "dataplex.referenceName"
+    And Replace input plugin property: "dataplexProjectID" with value: "dataplex.project"
+    And Enter input plugin property: "dataplexLocationID" with value: "dataplex.location"
+    And Enter input plugin property: "dataplexLakeID" with value: "dataplex.lake"
+    And Enter input plugin property: "dataplexZoneID" with value: "dataplex.zone"
+    And Enter input plugin property: "dataplexAssetID" with value: "dataplex.assetBQ"
+    And Enter input plugin property: "dataplexTableName" with value: "dataplex.outputTable"
+    And Click plugin property: "dataplexTruncateTable"
+    And Click plugin property: "dataplexUpdateTableSchema"
+    And Click plugin property: "dataplexRequirePartitionFilter"
+    And Validate "Dataplex" plugin properties
+
+  @BATCH-TS-DATAPLEX-DSGN-02
+  Scenario: To verify user should be able to validate fields with operation radio button, update table schema
+  toggle button and integer partitionin in dataplex sink with when asset type is selected as BigQuery dataset
```

Comment:
Please reword - not clear what this scenario description means

Reply:
Updated scenario description.